### DISCUSSION TITLE
intel i5 6 Gen ffmpeg hwaccel settings...

### DISCUSSION THREAD

**Discussion by Minglarn** [blakeblackshear/frigate#2315] [Ask A Question]
Hi... 
I'm trying to optimize my dedicated nuc (actally a HP ProDesk 400 G2) equipped with a Intel i5-6500T CPU. 
At the moment my settings for hwaccel is none...
Works well with avg cpu usage at 60-70% ... 
And as the GPU usage i none... 
intel_gpu_top ->
![image](https://user-images.githubusercontent.com/58695742/143096205-bbf6ebd1-bc17-407d-96db-cf684fb06ca8.png)

What is the best settings for this CPU? 
QSV or vaapi?
Need some advice as google is _not_ my best friend at the moment as it cant find any hwaccel settings for i5-6500T cpu.

**Comment by ozett** [comment]
for some testing?
https://github.com/blakeblackshear/frigate/issues/1997#issuecomment-975961986

some hints...
https://github.com/blakeblackshear/frigate/issues/1607#issuecomment-907786510

**Reply by Minglarn** [reply]
Thanks for the links, but ... when using qsv or vaapi my CPU usage goes up... Even if the GPU is working...
Must be something wrong in my ffmpeg settings... 
![image](https://user-images.githubusercontent.com/58695742/144087197-42563a35-3e8f-4404-b6bd-101b1bcb8cb0.png)
